assets/queries/openAPI/schema_object_incorrect_ref/query.rego

```diff
@@ -103,6 +104,7 @@ CxPolicy[result] {
 	openapi_lib.check_openapi(doc) != "undefined"
 
 	schema_ref := doc.components.responses[r].content[c].schema["$ref"]
+	openapi_lib.content_allowed("", r)
```

Comment:
I think this does not makes sense, since components does not need response code, as you can see [here](https://swagger.io/specification/#components-object)